src/google-maps/google-map/google-map.ts

```diff
@@ -78,6 +75,9 @@ export class GoogleMap implements OnChanges, OnInit, OnDestroy {
   private _mapEl: HTMLElement;
   _googleMap: UpdatedGoogleMap;
 
+  /** Whether we're currently rendering inside a browser. */
+  _isBrowser: boolean;
```

Comment:
Rather than adding an internal member to this class, we could add a standalone constant that isn't exported through `public-api.ts`

Reply:
We can't, because we resolve this through DI.